src/util/sss_iobuf.h

```diff
@@ -148,4 +193,14 @@ errno_t sss_iobuf_read_stringz(struct sss_iobuf *iobuf,
 
 errno_t sss_iobuf_write_stringz(struct sss_iobuf *iobuf,
                                 const char *str);
+
+errno_t sss_iobuf_read_string(TALLOC_CTX *mem_ctx,
+                              struct sss_iobuf *iobuf,
+                              char **_out,
+                              size_t *_len);
+
+errno_t sss_iobuf_write_string(struct sss_iobuf *iobuf,
```

Comment:
I don't find an implementation of those ^^ two functions.

Reply:
Good catch. Removed.